ocl/load-concepts-to-ocl.sh

```diff
@@ -189,11 +198,33 @@ create_hl7_medication_dispense_status_reason_source_in_ocl() {
       $OCL_API_URL/orgs/PIH/sources/
 }
 
+create_pih_malawi_source_in_ocl() {
+  curl --silent \
+      -H "Authorization: Token $OCL_API_TOKEN" \
+      -H "Accept: application/json" \
+      -H "Content-Type: application/json" \
+      --request POST \
+      --data '{"id":"PIH-Malawi","short_code":"PIHMalawi","name":"PIH Malawi","description":"(2015 or older) Partners In Health Malawi concept dictionary","source_type":"External","default_locale":"en","supported_locales":"en"}' \
+      $OCL_API_URL/orgs/PIH/sources/
+}
+
+create_liberia_moh_source_in_ocl() {
+  curl --silent \
+      -H "Authorization: Token $OCL_API_TOKEN" \
+      -H "Accept: application/json" \
+      -H "Content-Type: application/json" \
+      --request POST \
+      --data '{"id":"LiberiaMOH","short_code":"LiberiaMOH","name":"Liberia MoH","full_name":"LiberiaMOH","description":"Liberia Ministry of Health disease codes","source_type":"External","default_locale":"en","supported_locales":"en"}' \
+      $OCL_API_URL/orgs/PIH/sources/
+}
+
 # TODO: Update to the HL7 organization and/or remove HL7-MedicationDispenseStatusReason when added to OCL
+# TODO: After merge of Mike's code change mseaton to OpenConceptLab repo
 export_concepts_to_json() {
   pushd ${CODE_DIR}
   rm -fR ocl_omrs
   git clone https://github.com/OpenConceptLab/ocl_omrs.git
+# git clone https://github.com/mseaton/ocl_omrs.git
```

Comment:
You don't need to add this commented out code, it's no longer needed and shouldn't ever be needed again